meson.build

```diff
@@ -3434,6 +3434,15 @@ executable(
         install : (conf.get('HAVE_SYSV_COMPAT') == 1),
         install_dir : rootlibexecdir)
 
+executable(
+        'systemd-iocost',
```

Comment:
like the other udev helpers, should probably be placed in a subdir of src/udev/. i.e. compare with fido_id and similar.